src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -633,6 +633,7 @@ private function registerWorkflowConfiguration(array $workflows, ContainerBuilde
                     new Reference('security.authorization_checker'),
                     new Reference('security.authentication.trust_resolver'),
                     new Reference('security.role_hierarchy'),
+                    new Reference('validator'),
```

Comment:
this will fail when the Validator component is not installed or the `validation` section is not enabled